client/hotrod-client/src/main/java/org/infinispan/client/hotrod/impl/protocol/Codec27.java

```diff
@@ -23,7 +25,6 @@ public HeaderParams writeHeader(ByteBuf buf, HeaderParams params) {
          IntSet segments, int batchSize) {
       return new IteratorMapper<>(remoteCache.retrieveEntries(
             // Use the ToEmptyBytesKeyValueFilterConverter to remove value payload
-            "org.infinispan.server.hotrod.HotRodServer$ToEmptyBytesKeyValueFilterConverter", segments, batchSize),
-            e -> (K) e.getKey());
+            EMPTY_VALUE_CONVERTER, segments, batchSize), e -> (K) e.getKey());
```

Comment:
The comment still references the class name, but that doesn't make sense now that the class name was moved to a constant. I suggest `Use a converter to remove the value payload` instead.